frameworks/helloworld/tests/test_soak.py

```diff
@@ -1,11 +1,91 @@
 import pytest
+from shakedown import *
```

Comment:
Can we please not use this pattern at all anymore? When we do 'from X import *` it makes it completely impossible to use a good linter or know where a given imported function actually came from.

I realize this pattern was set before, but its time to change the precedent and we gotta start somewhere. :D